Если бы класс `ArticlePublishedException` не был потомком `ArticleException`, который указан как класс-исключение в `Article`
